pkg/member/member_control.go

```diff
@@ -317,11 +317,14 @@ func (m *memberControl) IsLearnerPresent(ctx context.Context) (bool, error) {
 
 // IsClusterScaledUp determines whether a etcd cluster is getting scale-up or not and returns a boolean
 func (m *memberControl) IsClusterScaledUp(ctx context.Context, clientSet client.Client) (bool, error) {
-	state, err := miscellaneous.GetInitialClusterStateIfScaleup(ctx, m.logger, clientSet, m.podName, m.podNamespace)
+	m.logger.Info("Checking whether etcd cluster is marked for scale-up")
+	etcdsts, err := miscellaneous.GetStatefulSet(ctx, clientSet, m.podNamespace, m.podName)
 	if err != nil {
-		m.logger.Errorf("annotation: %v is not present: %v", miscellaneous.ScaledToMultiNodeAnnotationKey, err)
```

Comment:
Can you re-add this log? seems useful to me

Reply:
previously we are calling `GetInitialClusterStateIfScaleup()` but now we are getting statefulset separately and `IsAnnotationPresent()` only returns boolean.

Reply:
If annotation is not present it can be infer from these 2 logs:
`Checking whether etcd cluster is marked for scale-up`
`Checking if member %s is part of a running cluster`

Reply:
Okay, now that I see the logs together, it seems fine enough